
base:
  wheel_diameter: 0.25
  L: 0.25  # FL wheel x coord
  W: 0.25  # FL wheel y coord
  d: 0  # deviation
  steer_theta_min: -180.0
  steer_theta_max: 180.0

laser:
  min_height: -0.4
  max_height: 1.0
  angle_min: -3.14
  angle_max: 3.14
  angle_increment: 0.0045 ##1440
  scan_time: 0.1
  range_min: 0.44
  range_max: 20.0


global_costmap:
  global_frame: map
  robot_base_frame: base_link
  update_frequency: 5.0
  static_map: true
  inflation_radius: 1.5
  resolution: 0.01

